chia/full_node/mempool_manager.py

```diff
@@ -254,15 +383,18 @@ def always(bundle_name: bytes32) -> bool:
             item_inclusion_filter = always
 
         log.info(f"Starting to make block, max cost: {self.max_block_clvm_cost}")
-        spend_bundles, cost_sum, additions, removals = self.process_mempool_items(item_inclusion_filter)
-        if len(spend_bundles) == 0:
+        processed_spend_bundles, sigs, coin_spends, cost_sum, additions, removals = self.process_mempool_items(
```

Comment:
I think returning this many values from a function is a code smell. Can you think of ways to simplify this? Maybe the interface to this function can shift a little to require fewer return values.

For instance, it could return an `Optional[SpendBundle]` instead of `coin_spends`, `sigs` and `processed_spend_bundles`.
It could print the info log message instead of returning `cost_sum`.